pkg/http/http.go

```diff
@@ -226,7 +226,7 @@ func performRequest(req *http.Request, verifyTLS bool) (int, http.Header, []byte
 		resp, err := client.Do(req) // nosemgrep: trailofbits.go.invalid-usage-of-modified-variable.invalid-usage-of-modified-variable
 		if err != nil {
 			if resp != nil {
-				defer resp.Body.Close()
+				defer resp.Body.Close() // #nosec: G307
 			}
```

Comment:
I think ignoring this error is fine given that this is an HTTP request and not a file. Wanted to share this snippet in case we do want to handle this

https://github.com/securego/gosec/issues/512#issuecomment-675286833

Reply:
Great find 👍 I'd definitely prefer resolving the error, rather than ignoring it. Just push the change